schema/crdb/dbinit.sql

```diff
@@ -2556,6 +2556,21 @@ INSERT INTO omicron.public.tuf_generation (
     (TRUE, 1)
 ON CONFLICT DO NOTHING;
 
+-- Trusted TUF root roles, used to verify TUF repo signatures
+CREATE TABLE IF NOT EXISTS omicron.public.tuf_trust_root (
+    -- Identity metadata (resource)
+    id UUID PRIMARY KEY,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
+    time_deleted TIMESTAMPTZ,
+    root_role JSONB NOT NULL
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS tuf_trust_root_by_id
+ON omicron.public.tuf_trust_root (id)
+WHERE
+    time_deleted IS NULL;
+
```

Comment:
Can you add a comment that this is used for listing (paginating through) non-deleted roots?  It took me a second to realize why you needed this since the primary key already provides uniqueness and fast lookup.